apps/video/test/marionette/control_test.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+var Video = require('./lib/video'),
+    assert = require('assert');
+    
+marionette('video info view', function() {
+  var client = marionette.client({
+    profile: {
+      prefs: {
+        'device.storage.enabled': true,
+        'device.storage.testing': true,
+        'device.storage.prompt.testing': true
+      }
+    },
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+  var app, selectors, initialTime;
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
+    
+    // Add file into the videos directory
+    client.fileManager.add({
+      type: 'videos',
+      filePath: 'test_media/Movies/elephants-dream.webm'
```

Comment:
I've noticed seeking within this video can be problematic. If you're seeing long delays when seeking within this video, you can try './test_media/samples/Movies/meetthecubs.webm'